src/sidebar/build-thread.js

```diff
@@ -263,7 +263,7 @@ function hasVisibleChildren(thread) {
  * @typedef Options
  * @prop {string[]} [selected] - List of currently-selected annotation ids, from
  *       the data store
- * @prop {string[]} [forceVisible] - List of ids of annotations that have
+ * @prop {string[]} [forcedVisible] - List of ids of annotations that have
```

Comment:
`forcedVisible` is a better adjectival form than `forceVisible`. `forceVisible` is the verb form (and is appropriate for function naming).